rules.d/60-persistent-storage.rules.in

```diff
@@ -156,6 +156,18 @@ ENV{ID_WWN_WITH_EXTENSION}=="?*", SYMLINK+="disk/by-id/wwn-$env{ID_WWN_WITH_EXTE
 ENV{ID_PART_ENTRY_UUID}=="?*", SYMLINK+="disk/by-partuuid/$env{ID_PART_ENTRY_UUID}"
 ENV{ID_PART_ENTRY_SCHEME}=="gpt", ENV{ID_PART_ENTRY_NAME}=="?*", SYMLINK+="disk/by-partlabel/$env{ID_PART_ENTRY_NAME}"
 
+# by-path/<path>/by-* links (path + partition/filesystem metadata)
+ENV{ID_PATH}!="?*", ENV{DEVTYPE}!="partition", GOTO="persistent_storage_by-path_parts_end"
```

Comment:
The first two conditions must be ORed, but udev rules always make conditions ANDed. So,
```
ENV{ID_PATH}=="", GOTO="persistent_storage_by-path_parts_end"
ENV{DEVTYPE}!="partition", GOTO="persistent_storage_by-path_parts_end"
```

Reply:
Ah yes, thank you!